tests/base/src/test/java/org/keycloak/tests/admin/ClientTest.java

```diff
@@ -938,91 +918,91 @@ public void updateClientWithProtocolMapper() {
         if (client.isEnabled() != null) Assert.assertEquals(client.isEnabled(), storedClient.isEnabled());
-        if (client.isAlwaysDisplayInConsole() != null) Assert.assertEquals(client.isAlwaysDisplayInConsole(), storedClient.isAlwaysDisplayInConsole());
+        if (client.isAlwaysDisplayInConsole() != null)
+            Assert.assertEquals(client.isAlwaysDisplayInConsole(), storedClient.isAlwaysDisplayInConsole());
         if (client.isBearerOnly() != null) Assert.assertEquals(client.isBearerOnly(), storedClient.isBearerOnly());
-        if (client.isPublicClient() != null) Assert.assertEquals(client.isPublicClient(), storedClient.isPublicClient());
-        if (client.isFullScopeAllowed() != null) Assert.assertEquals(client.isFullScopeAllowed(), storedClient.isFullScopeAllowed());
+        if (client.isPublicClient() != null)
+            Assert.assertEquals(client.isPublicClient(), storedClient.isPublicClient());
+        if (client.isFullScopeAllowed() != null)
+            Assert.assertEquals(client.isFullScopeAllowed(), storedClient.isFullScopeAllowed());
         if (client.getRootUrl() != null) Assert.assertEquals(client.getRootUrl(), storedClient.getRootUrl());
         if (client.getAdminUrl() != null) Assert.assertEquals(client.getAdminUrl(), storedClient.getAdminUrl());
         if (client.getBaseUrl() != null) Assert.assertEquals(client.getBaseUrl(), storedClient.getBaseUrl());
-        if (client.isSurrogateAuthRequired() != null) Assert.assertEquals(client.isSurrogateAuthRequired(), storedClient.isSurrogateAuthRequired());
-        if (client.getClientAuthenticatorType() != null) Assert.assertEquals(client.getClientAuthenticatorType(), storedClient.getClientAuthenticatorType());
+        if (client.isSurrogateAuthRequired() != null)
+            Assert.assertEquals(client.isSurrogateAuthRequired(), storedClient.isSurrogateAuthRequired());
+        if (client.getClientAuthenticatorType() != null)
+            Assert.assertEquals(client.getClientAuthenticatorType(), storedClient.getClientAuthenticatorType());
         if (client.getSecret() != null) Assert.assertEquals(client.getSecret(), storedClient.getSecret());
 
         if (client.getNotBefore() != null) {
-            Assert.assertEquals(client.getNotBefore(), storedClient.getNotBefore());
+            Assertions.assertEquals(client.getNotBefore(), storedClient.getNotBefore());
         }
         if (client.getDefaultRoles() != null) {
-            Set<String> set = new HashSet<String>();
-            for (String val : client.getDefaultRoles()) {
-                set.add(val);
-            }
-            Set<String> storedSet = new HashSet<String>();
-            for (String val : storedClient.getDefaultRoles()) {
-                storedSet.add(val);
-            }
+            Set<String> set = Set.of(client.getDefaultRoles());
+            Set<String> storedSet = Set.of(storedClient.getDefaultRoles());
```

Comment:
```suggestion
            Set<String> set = new HashSet<>(client.getDefaultRoles());
            Set<String> storedSet = new HashSet<>(storedClient.getDefaultRoles());
```

Reply:
Unfortunately, there is no Set constructor accepting a String array